src/components/MapDisplay.js

```diff
@@ -43,18 +44,18 @@ class MapDisplay extends React.Component {
   	this.setState({
   		heatmapCoords: [ ...this.state.heatmapCoords, {lat, lng}]
   	})
-    if (this._googleMap !== undefined) {      
+    if (this._googleMap !== undefined) {
       const point = new google.maps.LatLng(lat, lng)
       this._googleMap.heatmap.data.push(point)
     }
   }
-  toggleHeatMap() {    
+  toggleHeatMap() {
```

Comment:
Why are you altering these lines for no reason?

Reply:
I'm not quite sure. I don't even think I touched those lines. Perhaps it was my linter?

Reply:
Sorry about that I'll see if I can revert

Reply:
I'm pretty sure that was ESLint removing trailing whitespace at the end of lines

Reply:
It looks like all Nick did was remove a a few spaces, Definetly a linter.